site/posts/intake-esm-2020-3-16.ipynb

```diff
@@ -0,0 +1,781 @@
+    "            \"variable\":\"FLNS\",\n",
+    "            \"path\":\"s3://ncar-cesm-lens/atm/daily/cesmLE-20C-FLNS.zarr\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"component\":\"atm\",\n",
+    "            \"frequency\":\"daily\",\n",
+    "            \"experiment\":\"20C\",\n",
+    "            \"variable\":\"FLNSC\",\n",
+    "            \"path\":\"s3://ncar-cesm-lens/atm/daily/cesmLE-20C-FLNSC.zarr\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"component\":\"atm\",\n",
+    "            \"frequency\":\"daily\",\n",
+    "            \"experiment\":\"20C\",\n",
+    "            \"variable\":\"FLUT\",\n",
+    "            \"path\":\"s3://ncar-cesm-lens/atm/daily/cesmLE-20C-FLUT.zarr\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"component\":\"atm\",\n",
+    "            \"frequency\":\"daily\",\n",
+    "            \"experiment\":\"20C\",\n",
+    "            \"variable\":\"FSNS\",\n",
+    "            \"path\":\"s3://ncar-cesm-lens/atm/daily/cesmLE-20C-FSNS.zarr\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"component\":\"atm\",\n",
+    "            \"frequency\":\"daily\",\n",
+    "            \"experiment\":\"20C\",\n",
+    "            \"variable\":\"FSNSC\",\n",
+    "            \"path\":\"s3://ncar-cesm-lens/atm/daily/cesmLE-20C-FSNSC.zarr\"\n",
+    "        }\n",
+    "    ],\n",
+    "    \"attributes\":[\n",
+    "        {\n",
+    "            \"column_name\":\"component\",\n",
+    "            \"vocabulary\":\"\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"column_name\":\"frequency\",\n",
+    "            \"vocabulary\":\"\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"column_name\":\"experiment\",\n",
+    "            \"vocabulary\":\"\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"column_name\":\"variable\",\n",
+    "            \"vocabulary\":\"\"\n",
+    "        }\n",
+    "    ],\n",
+    "    \"assets\":{\n",
+    "        \"column_name\":\"path\",\n",
+    "        \"format\":\"zarr\"\n",
+    "    },\n",
+    "    \"aggregation_control\":{\n",
+    "        \"variable_column_name\":\"variable\",\n",
+    "        \"groupby_attrs\":[\n",
+    "            \"component\",\n",
+    "            \"experiment\",\n",
+    "            \"frequency\"\n",
+    "        ],\n",
+    "        \"aggregations\":[\n",
+    "            {\n",
+    "                \"type\":\"union\",\n",
+    "                \"attribute_name\":\"variable\",\n",
+    "                \"options\":{\n",
+    "                    \"compat\":\"override\"\n",
+    "                }\n",
+    "            }\n",
+    "        ]\n",
+    "    }\n",
+    "}\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "```python\n",
+    "import intake\n",
+    "col = intake.open_esm_datastore(\"catalog-dict-records.json\")\n",
+    "```"
+   ]
+  },
```

Comment:
Some text before this codeblock would be helpful to understand what you are demonstrating.